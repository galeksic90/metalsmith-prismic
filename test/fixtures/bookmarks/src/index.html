---
template: index.hbt
prismic:
  about:
    bookmark: 'about'
  jobs:
    bookmark: 'jobs'
  stores:
    bookmark: 'stores'
  missing:
    bookmark: 'missing'
---
