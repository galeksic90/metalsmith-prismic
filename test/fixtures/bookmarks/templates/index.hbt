<html>
<body>
    <div class="about">
        {{{ prismic.about.data.title.html }}}
        {{{ prismic.about.data.short_lede.html }}}
    </div>
    <div class="jobs">
        {{{ prismic.jobs.data.title.html }}}
        {{{ prismic.jobs.data.short_lede.html }}}
    </div>
    <div class="stores">
        {{{ prismic.stores.data.title.html }}}
        {{{ prismic.stores.data.short_lede.html }}}
    </div>
    <div class="missing">
        {{{ prismic.missing.data.title.html }}}
        {{{ prismic.missing.data.short_lede.html }}}
    </div>
</body>
</html>
