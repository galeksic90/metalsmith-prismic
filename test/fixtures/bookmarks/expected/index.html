<html>
<body>
    <div class="about">
        <h1>About us</h1>
        <p>Making great pastry is our passion. Turning them into people's delight is our commitment. Understand our values and learn to know us.</p>
    </div>
    <div class="jobs">
        <h1>Be at the top of the fine pastry scene</h1>
        <p>Join us, and get involved with creating the finest pastries that can be humanly made, and providing them to our customers.</p>
    </div>
    <div class="stores">
        <h1>Don't be a stranger!</h1>
        <p>They say it's better to travel with friends; that's why our staff is always friendly when they introduce you to the finest flavor journeys.</p>
    </div>
    <div class="missing">
        
        
    </div>
</body>
</html>
